transcendence-app/src/swagger-main.ts

```diff
@@ -0,0 +1,27 @@
+import { VersioningType } from '@nestjs/common';
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { writeFileSync } from 'fs';
+import { AppModule } from './app.module';
+import * as yaml from 'yaml';
+import { setupApp } from './setup-app';
+
+async function bootstrap() {
```

Comment:
Why do we need to detach the swagger generation from the compilation of the transcendence app?

I was thinking if we could just keep the swagger generation on build (or development mode) and just add the generated `.yaml` to the `.gitignore` as you've done

Reply:
I guess we could dissassemble the root docker-compose and have a dependency on the webapp service delay it's booting up until the swagger-spec.yaml AND the rest of the generated files have been created by the transcendence-app service + maybe some intermediate service.

This solution seemed a bit simpler, and more modular, so this is how we decided to proceed.

Reply:
Let's discuss this in a different issue, please.